package/MDAnalysis/core/universe.py

```diff
@@ -738,10 +743,18 @@ def __repr__(self):
             n_atoms=len(self.atoms))
 
     def __getstate__(self):
-        raise NotImplementedError
+        # Universe's two "legs" of topology and traj both serialise themselves
+        # the only other state held in Universe is anchor name?
+        return self.anchor_name, self._topology, self._trajectory
+
+    def __setstate__(self, args):
+        self._anchor_name = args[0]
```

Comment:
use the `anchor_name` property rather than doing it manually here

Reply:
The problem about using `anchor_name` here is that `anchor_name` have to first remove anchor, which will raise an error because the new universe doesn't have one.
So it might becomes:
```python
self._anchor_name = ""
self.anchor_name = args[0]
```

Reply:
is this OK @richardjgowers ?